testing/containers/ceph/Dockerfile

```diff
@@ -9,6 +9,7 @@ RUN true && \
     yum install -y \
         git wget curl make \
         /usr/bin/cc /usr/bin/c++ \
+        rbd-fuse \
```

Comment:
It'd be nice if we didn't need to add a new dependency for this, but I won't complain too loudly, since we can always come back and revisit it later.

Reply:
Well, if you know another way to test that mirroring is configured correctly, I'm happy to hear it. Or do you suggest not to test it at all?